lib/util/support.js

```diff
@@ -126,6 +126,12 @@ function mobileWebKitBrowser(browser) {
   }) || null;
 }
 
+// Browser detection to check for whether or not the browser is Chrome iOS.
+// eslint-disable-next-line no-warning-comments
+// TODO(joyce): Move browser detection to a different util
+const isAndroid = /android/.test(navigator.userAgent);
+const isChromeIOS = guessBrowser() === 'chrome' && isMobile() && !isAndroid;
```

Comment:
is there anything in chrome ios useragent that we can use to see if its on iOS...that is instead of assuming its 
`iOS = !isAndroid`

Reply:
I think we can do `/iPad|iPhone|iPod/`

Reply:
Please add unit tests for isChromeIOS once we make it a function.

Reply:
Sounds good, I'll go ahead and make that change. 

@charliesantos @makarandp0 I'll make another unit test file just for these browser device detection utils since we're going to be moving them out of the support.js file. How does that sound?

Reply:
Sounds good to me.